nimble/host/audio/services/bass/src/ble_audio_svc_bass.c

```diff
@@ -326,6 +361,7 @@ ble_svc_audio_bass_add_source(uint8_t *data, uint16_t data_len, uint16_t conn_ha
             ev.bass_operation_status.status = BLE_HS_EREJECT;
             goto done;
         }
+        operation.add_source.bis_sync[i] = get_le32(&data[offset]);
         operation.add_source.subgroups[i].bis_sync_state = get_le32(&data[offset]);
```

Comment:
(1) here. See, the `&data[offset]` is passed in two variables.

Reply:
Ok, removed.